cmd/system-controller/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/skupperproject/skupper/internal/nonkube/controller"
+	"github.com/skupperproject/skupper/internal/version"
+	"github.com/skupperproject/skupper/pkg/nonkube/api"
+)
+
+func main() {
+	parseFlags()
+	log.Printf("Version: %s", version.Version)
```

Comment:
should the system controller version be included in the images.go file like the rest of components?

Reply:
Nice catch @nluaces ! I will add them.